lib/inspec/base_cli.rb

```diff
@@ -91,11 +91,18 @@ def self.default_options
       }
     end
 
-    def self.parse_reporters(opts) # rubocop:disable Metrics/AbcSize
+    def self.parse_reporters(opts) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
       # merge in any legacy formats as reporter
       # this method will only be used for ad-hoc runners
       if !opts['format'].nil? && opts['reporter'].nil?
         warn '[DEPRECATED] The option --format is being is being deprecated and will be removed in inspec 3.0. Please use --reporter'
+
+        # see if we are using the legacy output to write to files
+        if opts['output']
+          opts['format'] = "#{opts['format']}:#{opts['output']}"
```

Comment:
I think we should print that this is deprecated and will be removed in inspec 3.0

Reply:
Added: